### PR TITLE
release docs

### PR DESCRIPTION
**Pull request by jyecusch**

### CODE REVIEW

**Review by raksiv — DISMISSED** [review]

**Review by davemooreuws — APPROVED** [review]

File: doc.config.json
Comment (davemooreuws) [inline doc.config.json:72]: This is not valid, wait until supabase PR is merged before merging this one.

**Review by davemooreuws — CHANGES_REQUESTED** [review]

File: doc.config.json
Comment (jyecusch) [reply doc.config.json]: merged now

**Review by davemooreuws — DISMISSED** [review]

**Review by tjholm — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Review by davemooreuws — APPROVED** [review]

**Pull request merged**
